assets/scss/main.css.scss

```diff
@@ -18,6 +18,10 @@
 
 $block-background-color: #fafafa;
 
+body {
+  -webkit-font-smoothing: antialiased;
+}
+
```

Comment:
I would suggest not including this. Font smoothing has been removed from both USWDS and the Login.gov Design System intentionally.

For history, see:

* https://github.com/18F/identity-design-system/pull/362#issuecomment-1699263924
* https://github.com/uswds/uswds/pull/5250

```suggestion
```

Reply:
Thanks for this history, @aduth, I found this super helpful.

I pulled this in because it's currently active in https://github.com/GSA-TTS/identity-site/blob/a2a42a1d00d1d58dc5c45c5efaa4cbf7aba00898/_sass/components/_typography.scss#L8

I personally think the text is slightly easier to read without it, but I assumed consistency was more important.

Reply:
Yeah, it probably ought to be removed from the brochure site as well.